src/server/server.go

```diff
@@ -29,6 +31,7 @@ import (
 //	@security					API Key
 func CreateApp(db *gorm.DB) *fiber.App {
 	app := fiber.New()
+	app.Use(logger.NewLogger())
```

Comment:
for now this is hardcoded to run even on prod right?

Reply:
yes is it, next we should discuss about log format.

Reply:
can we do this only for non production environments?

Reply:
sure, please review my latest commit @championswimmer